### ISSUE TITLE
cmd/gosky: plc flag not defined for didGet

### ISSUE THREAD

**Issue by karalabe** [bluesky-social/indigo#12]
Can I haz:

```
		&cli.StringFlag{
			Name:    "plc",
			Value:   "https://plc.staging.bsky.dev",
			EnvVars: []string{"BSKY_PLC_URL"},
		},
```

in https://github.com/bluesky-social/indigo/blob/main/cmd/gosky/main.go#L155, otherwise https://github.com/bluesky-social/indigo/blob/main/cmd/gosky/util/util.go#L24 is a noop and the command goes boom

```
$ go run ./cmd/gosky did get @karalabe.bsky.social
Get "/@karalabe.bsky.social": unsupported protocol scheme ""
exit status 1
```

**Comment by whyrusleeping** [comment]
Resolved in 420c09856883222833769670f0fc93a6a58aa9b2

```
why@sirius ~/c/gosky (main)> ./gosky did get did:plc:ragtjsm2j2vknwkz3zp4oxrd
{
  "@context": [
    "https://www.w3.org/ns/did/v1",
    "https://w3id.org/security/suites/secp256k1-2019/v1"
  ],
  "id": "did:plc:ragtjsm2j2vknwkz3zp4oxrd",
  "alsoKnownAs": [
    "https://paul.bsky.social"
  ],
  "authentication": null,
  "verificationMethod": [
    {
      "id": "#signingKey",
      "type": "EcdsaSecp256k1VerificationKey2019",
      "controller": "did:plc:ragtjsm2j2vknwkz3zp4oxrd",
      "publicKeyMultibase": "zMy9mFCjiJvSfUoboTdFZV4n2sxq1JpUWT8Xxkt16gNhBf6oW3FurYQLAYDtThJXD1rHtWmugFCtrQqGoJk1x5PpK"
    },
    {
      "id": "#recoveryKey",
      "type": "EcdsaSecp256k1VerificationKey2019",
      "controller": "did:plc:ragtjsm2j2vknwkz3zp4oxrd",
      "publicKeyMultibase": "zNET4n9SXotP16s46Dg3GPdWCL2iqe6MDEQ9DsT1PeTVSQgegPm4MChUmQW3A87cqaiSPa29PYGcZoKUaRZYoN4Fc"
    }
  ],
  "service": [
    {
      "id": "#atpPds",
      "type": "AtpPersonalDataServer",
      "serviceEndpoint": "https://bsky.social"
    }
  ]
}
```

**Comment by whyrusleeping** [comment]
Thanks for the report :)
